pkg/transforms/tags_test.go

```diff
@@ -142,7 +142,7 @@ func TestTags_AddGenericTags(t *testing.T) {
 			var continuePipeline bool
 			var result interface{}
 
-			target := NewGenericTags(testCase.TagsToAdd)
+			target := NewTags(testCase.TagsToAdd)
```

Comment:
Also change name of this test to `TestTags_AddTags`

Reply:
test name has been changed